src/shipit_signoff/shipit_signoff/api.yml

```diff
@@ -63,18 +94,44 @@ paths:
       responses:
         200:
           description: Removal of step
+          
+    get:
+      operationId: "shipit_signoff.api.get_step"
+      description: Step definition
+      parameters:
+          - $ref: '#/parameters/uid'
+      responses:
+        200:
+          description: Definition of step
+          schema:
+            $ref: '#/definitions/Step'
+
 
   # Signoff specific
-  /{uid}/signoff:
+  # User is acquired from the access token
+  # Do we want the user to claim a specific group?
+  /step/{uid}/sign:
     put:
-      operationId: "shipit_signoff.api.signoff"
+      operationId: "shipit_signoff.api.sign_off"
       description: "Sign off"
       parameters:
         - $ref: '#/parameters/uid'
       responses:
         200:
           description: signed off
 
+  /step/{uid}/unsign:
```

Comment:
How do you feel about using a DELETE on the sign endpoint for this instead? It feels a little cleaner to me.